.github/workflows/run_shark_tank.yml

```diff
@@ -0,0 +1,140 @@
+name: SHARK
+
+on:
+  # schedule:
+  #  - cron: '0 10 * * *'
+  pull_request:
```

Comment:
Do we want to trigger this workflow in the presubmit?

Reply:
It also looks like we allow presubmit runners to write to the SHARK report GCS bucket right now, put a reminder to remove that extra permission when we're ready to merge.

Reply:
This is now working. I've made updates to not trigger on a pull request and I've removed the presubmit permissions on the GCS bucket. PTAL.